source/calendar/calendar-styles.css

```diff
@@ -77,44 +77,133 @@ body {
+.current-day.february {
+    background: linear-gradient(135deg, pink, violet);
+} 
+.current-day.february:hover {
+    background: linear-gradient(135deg, violet, pink);
 }
 
-.day.winter {
-    color: #00f; 
+.march { color: #28a745; } 
+.day.march:hover { background-color: rgb(86, 234, 135);}  
+.current-day.march {
+    background: linear-gradient(135deg, rgb(82, 213, 161), rgb(78, 194, 36));
+} 
+.current-day.march:hover {
+    background: linear-gradient(135deg, rgb(78, 194, 36), rgb(82, 213, 161));
 }
 
-.month-color {
-    color: #333;
+.april { color: #e83e8c; } 
+.day.april:hover { background-color: #f476b1;} 
+.current-day.april {
+    background: linear-gradient(135deg, rgb(219, 57, 208), rgb(195, 24, 129));
+} 
+.current-day.april:hover {
+    background: linear-gradient(135deg, rgb(195, 24, 129), rgb(219, 57, 208));
+}
+
+.may { color: #790258; }  
+.day.may:hover { background-color: #ea65c4;} 
+.current-day.may {
+    background: linear-gradient(135deg, rgb(236, 55, 149), rgb(183, 17, 161));
+} 
+.current-day.may:hover {
+    background: linear-gradient(135deg, rgb(183, 17, 161), rgb(236, 55, 149));
+}
+  
+.june { color: #154a27; } 
+.day.june:hover { background-color: #47d175;}
+.current-day.june {
+    background: linear-gradient(135deg, rgb(101, 225, 142), rgb(61, 183, 17));
+} 
+.current-day.june:hover {
+    background: linear-gradient(135deg, rgb(61, 183, 17), rgb(101, 225, 142));
+}
+
+
+.july { color: #17a2b8; } 
+.day.july:hover { background-color: #5ed9ec;}  
+.current-day.july {
+    background: linear-gradient(135deg, rgb(21, 234, 227), rgb(17, 139, 183));
+} 
+.current-day.july:hover {
+    background: linear-gradient(135deg, rgb(17, 139, 183), rgb(21, 234, 227));
+}
+
+.august { color: #d1890c; } 
+.day.august:hover { background-color: #f5bb58;} 
+.current-day.august {
+    background: linear-gradient(135deg, rgb(236, 170, 38), rgb(199, 228, 8));
+} 
+.current-day.august:hover {
+    background: linear-gradient(135deg, rgb(199, 228, 8), rgb(236, 170, 38));
 }
 
-.january { color: #c14300; }  
-.february { color: #f29cac; } 
-.march { color: #28a745; }   
-.april { color: #e83e8c; }   
-.may { color: #790258; }     
-.june { color: #154a27; }    
-.july { color: #17a2b8; }     
-.august { color: #d1890c; }   
 .september { color: #dc3545; }
+.day.september:hover { background-color: #ed7480;}
+.current-day.september {
+    background: linear-gradient(135deg, rgb(201, 55, 30), rgb(252, 57, 67));
+} 
+.current-day.september:hover {
+    background: linear-gradient(135deg, rgb(252, 57, 67), rgb(201, 55, 30));
+}
+
 .october { color: #d45810; }  
+.day.october:hover { background-color: #e8864d;}
+.current-day.october {
+    background: linear-gradient(135deg, rgb(242, 214, 55), rgb(231, 151, 14));
+} 
+.current-day.october:hover {
+    background: linear-gradient(135deg, rgb(231, 151, 14), rgb(242, 214, 55));
+}
+
 .november { color: #343a40; } 
+.day.november:hover { background-color: #a6b6c6;}
+.current-day.november {
```

Comment:
Is there any overlap between the coloring for the two files? Might be good to move duplicate CSS to a separate file if possible. Otherwise should be fine.